docs/_docs/cross-chain/bridge.md

```diff
@@ -0,0 +1,62 @@
+---
+---
+# The Standard Obscuro Bridge
+
+The standard Obscuro bridge is a decentralized asset bridge that uses a wrapped token mint and burn pattern. Under the hood it is based on the cross chain messaging protocol and exists entirely as a smart contract without the need of separate runnables or nodes.
+
+## Contract Addresses
+
+## Interacting with the contract
+
+Users interact with the bridge contract using either the `sendERC20` or `sendNative` functions. In order to send ERC20 tokens they need to be whitelisted. 
+
+The pattern of interacting on both layers with `sendERC20` is as follows: 
+ 1. Approve the bridge contract to act on your behalf for X amount of tokens using the standard ERC20 `approve()` method.
+ 2. Call `sendERC20` with the amount approved and the address of the recipient on the other side.
+ 3. Wait a tiny bit for the cross chain protocol to synchronize the state of the layers.
+ 4. Use your assets on the other layer.
+
+Interacting with `sendNative` does not require approving an allowance, but instead just add value to the transaction (or evm call).
+The value received by the bridge contract during the execution of `sendNative` is what will be logged as transfer to the other layer.
+
+## Layer 1 To Layer 2 Specifics
+
+The `ObscuroBridge.sol` contract is responsible for managing the layer 1 side of the bridge. It's the "bridge to Obscuro".
+The `EthereumBridge.sol` contract is responsible for managing the layer 2 side of the bridge. It's the "bridge to Ethereum".
+
+In order to bridge tokens over they need to be whitelisted. **Only accounts with the admin role can whitelist tokens!**
```

Comment:
Do we / or can we mention here the process of requesting an ERC20 to be whitelisted if only accounts with the admin role can do this i.e. if I am a user of Obscuro and want to bridge my token, who do I ask to perform the whitelist?